vibe_code_apps_22/eplus_gym/parametric_daily_controller.py

```
@@ -0,0 +1,159 @@
+
+
+def build_htg_sp_series_f(params: ParametricDailyParams) -> List[float]:
+    """Emit 96 heating setpoints °F with optional recovery ramp into occupied SP."""
+    occ = float(params.occupied_heating_f) + float(params.occupied_setpoint_offset_f)
+    unocc = float(params.unoccupied_heating_f)
+    start = int(params.occupancy_start_step) % 96
+    end = int(params.occupancy_end_step) % 96
+    if end <= start:
+        end = min(96, start + 1)
+
+    lead_steps = max(0, int(round(params.recovery_start_minutes_before_occupancy / 15.0)))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)